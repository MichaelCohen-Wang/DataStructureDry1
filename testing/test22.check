add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
update_pirate_treasure: FAILURE
add_ship: SUCCESS
add_pirate: SUCCESS
add_pirate: SUCCESS
add_ship: SUCCESS
update_pirate_treasure: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
get_treasure: SUCCESS, 30
add_pirate: SUCCESS
add_pirate: SUCCESS
update_pirate_treasure: FAILURE
add_ship: SUCCESS
add_pirate: SUCCESS
remove_pirate: SUCCESS
update_pirate_treasure: SUCCESS
add_ship: SUCCESS
update_pirate_treasure: SUCCESS
remove_ship: SUCCESS
