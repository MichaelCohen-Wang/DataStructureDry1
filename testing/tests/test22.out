add_pirate: FAILURE
remove_pirate: FAILURE
add_ship: SUCCESS
remove_pirate: FAILURE
remove_ship: SUCCESS
update_pirate_treasure: FAILURE
add_ship: SUCCESS
update_pirate_treasure: FAILURE
add_ship: SUCCESS
get_cannons: SUCCESS, 21
get_treasure: FAILURE
update_pirate_treasure: FAILURE
add_pirate: SUCCESS
update_pirate_treasure: FAILURE
get_cannons: SUCCESS, 21
ships_battle: SUCCESS
remove_ship: SUCCESS
add_ship: SUCCESS
get_treasure: SUCCESS, 12
ships_battle: FAILURE
update_pirate_treasure: SUCCESS
add_pirate: SUCCESS
get_cannons: FAILURE
add_ship: SUCCESS
update_pirate_treasure: SUCCESS
add_ship: SUCCESS
get_richest_pirate: SUCCESS, 299
add_ship: SUCCESS
remove_ship: SUCCESS
update_pirate_treasure: SUCCESS
update_pirate_treasure: SUCCESS
add_pirate: SUCCESS
remove_pirate: SUCCESS
add_pirate: SUCCESS
ships_battle: FAILURE
ships_battle: SUCCESS
add_pirate: SUCCESS
add_pirate: INVALID_INPUT
ships_battle: SUCCESS
add_pirate: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
get_cannons: SUCCESS, 17
update_pirate_treasure: SUCCESS
update_pirate_treasure: SUCCESS
update_pirate_treasure: SUCCESS
update_pirate_treasure: SUCCESS
remove_pirate: SUCCESS
update_pirate_treasure: SUCCESS
add_ship: SUCCESS
update_pirate_treasure: SUCCESS
remove_pirate: SUCCESS
treason: SUCCESS
get_cannons: SUCCESS, 21
treason: SUCCESS
remove_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
treason: FAILURE
get_cannons: SUCCESS, 13
add_pirate: SUCCESS
add_ship: SUCCESS
get_treasure: SUCCESS, 69
get_treasure: SUCCESS, 48
remove_pirate: SUCCESS
update_pirate_treasure: SUCCESS
remove_ship: SUCCESS
add_pirate: SUCCESS
add_pirate: SUCCESS
get_richest_pirate: SUCCESS, 129
get_cannons: FAILURE
add_pirate: FAILURE
add_pirate: FAILURE
remove_pirate: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
remove_pirate: FAILURE
remove_ship: SUCCESS
get_cannons: SUCCESS, 21
add_pirate: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
treason: FAILURE
treason: FAILURE
remove_pirate: SUCCESS
treason: SUCCESS
update_pirate_treasure: FAILURE
add_pirate: SUCCESS
get_cannons: SUCCESS, 2
add_ship: SUCCESS
get_richest_pirate: FAILURE
add_pirate: SUCCESS
add_pirate: SUCCESS
add_pirate: FAILURE
add_ship: SUCCESS
add_pirate: SUCCESS
add_ship: SUCCESS
remove_ship: SUCCESS
add_pirate: FAILURE
get_richest_pirate: SUCCESS, 377
add_ship: SUCCESS
remove_pirate: SUCCESS
treason: FAILURE
remove_pirate: SUCCESS
get_treasure: SUCCESS, 6
ships_battle: SUCCESS
add_ship: SUCCESS
get_cannons: SUCCESS, 10
get_richest_pirate: FAILURE
update_pirate_treasure: SUCCESS
remove_ship: FAILURE
ships_battle: FAILURE
ships_battle: FAILURE
ships_battle: SUCCESS
get_richest_pirate: FAILURE
get_treasure: INVALID_INPUT
add_ship: SUCCESS
ships_battle: FAILURE
remove_ship: FAILURE
add_ship: SUCCESS
ships_battle: SUCCESS
treason: FAILURE
ships_battle: SUCCESS
ships_battle: SUCCESS
add_pirate: SUCCESS
add_ship: SUCCESS
get_richest_pirate: SUCCESS, 377
remove_ship: SUCCESS
add_pirate: SUCCESS
get_treasure: SUCCESS, -1
update_pirate_treasure: SUCCESS
remove_ship: SUCCESS
ships_battle: FAILURE
add_ship: SUCCESS
get_cannons: SUCCESS, 23
add_pirate: SUCCESS
ships_battle: FAILURE
remove_pirate: SUCCESS
ships_battle: SUCCESS
ships_battle: FAILURE
get_cannons: SUCCESS, 23
add_ship: SUCCESS
get_cannons: SUCCESS, 0
ships_battle: SUCCESS
remove_ship: FAILURE
get_treasure: FAILURE
get_richest_pirate: SUCCESS, 204
remove_ship: FAILURE
get_cannons: FAILURE
update_pirate_treasure: FAILURE
treason: FAILURE
update_pirate_treasure: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
treason: FAILURE
remove_pirate: SUCCESS
get_richest_pirate: SUCCESS, 129
remove_pirate: SUCCESS
add_pirate: SUCCESS
remove_pirate: INVALID_INPUT
ships_battle: SUCCESS
remove_ship: SUCCESS
add_pirate: SUCCESS
get_cannons: FAILURE
get_cannons: SUCCESS, 8
add_ship: SUCCESS
get_treasure: FAILURE
get_cannons: SUCCESS, 10
ships_battle: SUCCESS
remove_ship: SUCCESS
get_richest_pirate: SUCCESS, 72
remove_ship: SUCCESS
add_pirate: SUCCESS
remove_ship: FAILURE
update_pirate_treasure: FAILURE
get_cannons: SUCCESS, 18
add_ship: SUCCESS
add_pirate: FAILURE
add_pirate: SUCCESS
get_richest_pirate: SUCCESS, 482
add_ship: SUCCESS
add_pirate: SUCCESS
remove_pirate: SUCCESS
remove_ship: SUCCESS
get_richest_pirate: SUCCESS, 10
remove_pirate: FAILURE
treason: FAILURE
add_ship: SUCCESS
get_treasure: SUCCESS, 17
get_cannons: SUCCESS, 10
remove_ship: FAILURE
get_treasure: FAILURE
add_ship: SUCCESS
get_treasure: FAILURE
get_cannons: SUCCESS, 22
add_ship: SUCCESS
get_cannons: SUCCESS, 8
remove_ship: FAILURE
get_cannons: SUCCESS, 22
get_richest_pirate: FAILURE
add_ship: SUCCESS
ships_battle: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
get_cannons: SUCCESS, 20
ships_battle: FAILURE
get_richest_pirate: FAILURE
get_cannons: FAILURE
get_treasure: FAILURE
get_richest_pirate: SUCCESS, 10
get_treasure: SUCCESS, 18
remove_pirate: SUCCESS
remove_ship: SUCCESS
add_pirate: SUCCESS
add_pirate: FAILURE
add_ship: SUCCESS
get_cannons: FAILURE
get_richest_pirate: FAILURE
add_ship: SUCCESS
ships_battle: SUCCESS
get_cannons: SUCCESS, 2
update_pirate_treasure: FAILURE
get_cannons: FAILURE
add_ship: SUCCESS
remove_pirate: SUCCESS
ships_battle: FAILURE
add_pirate: SUCCESS
remove_ship: FAILURE
get_richest_pirate: FAILURE
add_ship: SUCCESS
get_cannons: FAILURE
add_ship: SUCCESS
remove_ship: FAILURE
get_cannons: SUCCESS, 20
get_cannons: SUCCESS, 8
get_cannons: SUCCESS, 11
get_treasure: FAILURE
remove_ship: SUCCESS
get_richest_pirate: SUCCESS, 72
get_cannons: SUCCESS, 8
treason: FAILURE
remove_ship: FAILURE
add_ship: SUCCESS
get_cannons: SUCCESS, 23
add_pirate: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
get_cannons: SUCCESS, 8
get_cannons: SUCCESS, 0
remove_pirate: SUCCESS
add_ship: SUCCESS
remove_pirate: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
get_treasure: SUCCESS, -1
add_ship: SUCCESS
add_pirate: SUCCESS
ships_battle: FAILURE
treason: FAILURE
get_cannons: SUCCESS, 22
add_ship: SUCCESS
remove_ship: SUCCESS
update_pirate_treasure: FAILURE
update_pirate_treasure: FAILURE
ships_battle: FAILURE
get_richest_pirate: SUCCESS, 352
add_pirate: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
update_pirate_treasure: SUCCESS
add_pirate: FAILURE
add_pirate: FAILURE
get_treasure: FAILURE
treason: SUCCESS
get_richest_pirate: FAILURE
add_ship: SUCCESS
add_pirate: SUCCESS
get_treasure: SUCCESS, 7
ships_battle: SUCCESS
treason: FAILURE
get_treasure: FAILURE
get_cannons: SUCCESS, 8
treason: FAILURE
treason: FAILURE
get_treasure: FAILURE
treason: SUCCESS
remove_ship: FAILURE
get_cannons: FAILURE
get_richest_pirate: FAILURE
remove_pirate: FAILURE
add_ship: SUCCESS
remove_ship: SUCCESS
remove_ship: SUCCESS
remove_pirate: SUCCESS
get_richest_pirate: SUCCESS, 482
update_pirate_treasure: SUCCESS
get_richest_pirate: FAILURE
add_ship: SUCCESS
add_pirate: SUCCESS
get_treasure: FAILURE
remove_pirate: FAILURE
treason: FAILURE
treason: SUCCESS
get_richest_pirate: FAILURE
add_pirate: SUCCESS
add_ship: SUCCESS
get_cannons: FAILURE
remove_ship: SUCCESS
add_pirate: FAILURE
ships_battle: FAILURE
treason: SUCCESS
add_ship: SUCCESS
get_richest_pirate: FAILURE
remove_ship: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
get_treasure: SUCCESS, 18
update_pirate_treasure: SUCCESS
add_pirate: FAILURE
get_richest_pirate: FAILURE
remove_ship: SUCCESS
get_cannons: SUCCESS, 21
add_ship: SUCCESS
get_cannons: FAILURE
get_richest_pirate: FAILURE
get_cannons: SUCCESS, 13
treason: FAILURE
ships_battle: FAILURE
get_richest_pirate: SUCCESS, 287
ships_battle: FAILURE
get_cannons: SUCCESS, 14
get_cannons: SUCCESS, 25
add_ship: SUCCESS
get_cannons: FAILURE
add_pirate: SUCCESS
add_ship: SUCCESS
ships_battle: FAILURE
add_ship: SUCCESS
get_richest_pirate: FAILURE
get_richest_pirate: FAILURE
remove_ship: FAILURE
add_pirate: FAILURE
remove_pirate: SUCCESS
remove_pirate: FAILURE
get_richest_pirate: SUCCESS, 219
add_ship: SUCCESS
get_cannons: SUCCESS, 29
add_ship: SUCCESS
get_treasure: FAILURE
treason: FAILURE
remove_pirate: FAILURE
get_treasure: FAILURE
get_richest_pirate: SUCCESS, 507
remove_ship: FAILURE
get_cannons: SUCCESS, 22
remove_ship: FAILURE
get_treasure: SUCCESS, 13
remove_ship: SUCCESS
remove_pirate: SUCCESS
update_pirate_treasure: SUCCESS
get_cannons: SUCCESS, 0
remove_pirate: FAILURE
get_richest_pirate: FAILURE
get_richest_pirate: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
remove_pirate: SUCCESS
ships_battle: SUCCESS
add_ship: SUCCESS
get_cannons: FAILURE
get_richest_pirate: FAILURE
add_pirate: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
get_treasure: FAILURE
ships_battle: FAILURE
remove_ship: SUCCESS
remove_pirate: FAILURE
get_richest_pirate: FAILURE
get_cannons: SUCCESS, 10
ships_battle: SUCCESS
ships_battle: FAILURE
get_cannons: SUCCESS, 30
add_ship: SUCCESS
get_richest_pirate: FAILURE
ships_battle: SUCCESS
remove_ship: SUCCESS
add_pirate: SUCCESS
add_pirate: FAILURE
get_cannons: SUCCESS, 21
add_pirate: SUCCESS
get_richest_pirate: FAILURE
get_cannons: SUCCESS, 23
get_treasure: SUCCESS, 8
add_pirate: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
add_pirate: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
treason: FAILURE
get_treasure: FAILURE
ships_battle: FAILURE
get_cannons: FAILURE
ships_battle: SUCCESS
ships_battle: FAILURE
get_cannons: SUCCESS, 15
update_pirate_treasure: FAILURE
update_pirate_treasure: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
remove_pirate: FAILURE
remove_ship: SUCCESS
get_cannons: SUCCESS, 18
get_treasure: SUCCESS, 27
update_pirate_treasure: SUCCESS
remove_pirate: FAILURE
add_pirate: FAILURE
get_cannons: SUCCESS, 22
ships_battle: FAILURE
add_pirate: SUCCESS
get_cannons: SUCCESS, 11
remove_pirate: SUCCESS
remove_ship: FAILURE
update_pirate_treasure: FAILURE
ships_battle: SUCCESS
get_cannons: SUCCESS, 10
ships_battle: SUCCESS
get_richest_pirate: FAILURE
remove_ship: FAILURE
get_cannons: FAILURE
get_richest_pirate: FAILURE
update_pirate_treasure: SUCCESS
get_treasure: SUCCESS, 23
treason: FAILURE
get_richest_pirate: FAILURE
remove_ship: SUCCESS
add_ship: SUCCESS
get_richest_pirate: FAILURE
remove_pirate: SUCCESS
get_treasure: FAILURE
get_richest_pirate: FAILURE
treason: FAILURE
get_cannons: FAILURE
add_pirate: SUCCESS
remove_ship: SUCCESS
ships_battle: SUCCESS
add_pirate: FAILURE
ships_battle: FAILURE
ships_battle: FAILURE
remove_ship: SUCCESS
ships_battle: FAILURE
treason: SUCCESS
update_pirate_treasure: FAILURE
add_pirate: SUCCESS
treason: FAILURE
remove_ship: SUCCESS
ships_battle: SUCCESS
remove_pirate: SUCCESS
treason: FAILURE
treason: FAILURE
treason: FAILURE
treason: FAILURE
get_cannons: SUCCESS, 13
ships_battle: FAILURE
ships_battle: SUCCESS
remove_ship: SUCCESS
add_pirate: SUCCESS
get_richest_pirate: FAILURE
ships_battle: FAILURE
add_pirate: SUCCESS
remove_pirate: FAILURE
get_cannons: SUCCESS, 26
get_cannons: SUCCESS, 2
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
get_richest_pirate: FAILURE
remove_pirate: FAILURE
remove_ship: SUCCESS
get_cannons: SUCCESS, 10
add_ship: SUCCESS
remove_ship: SUCCESS
add_pirate: SUCCESS
add_pirate: SUCCESS
remove_pirate: FAILURE
add_ship: SUCCESS
get_richest_pirate: FAILURE
update_pirate_treasure: SUCCESS
update_pirate_treasure: SUCCESS
remove_pirate: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
add_ship: SUCCESS
get_cannons: SUCCESS, 23
ships_battle: SUCCESS
treason: SUCCESS
treason: FAILURE
get_cannons: SUCCESS, 10
add_ship: SUCCESS
get_cannons: SUCCESS, 4
remove_pirate: SUCCESS
remove_ship: SUCCESS
remove_pirate: SUCCESS
add_ship: SUCCESS
get_treasure: SUCCESS, 20
remove_pirate: FAILURE
get_richest_pirate: FAILURE
treason: FAILURE
treason: FAILURE
update_pirate_treasure: FAILURE
add_pirate: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
remove_ship: SUCCESS
add_ship: SUCCESS
remove_pirate: FAILURE
get_cannons: SUCCESS, 4
get_cannons: SUCCESS, 16
update_pirate_treasure: FAILURE
remove_ship: SUCCESS
add_pirate: FAILURE
add_ship: SUCCESS
get_treasure: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
get_cannons: SUCCESS, 10
remove_ship: SUCCESS
remove_ship: SUCCESS
treason: SUCCESS
get_cannons: SUCCESS, 30
remove_ship: FAILURE
add_ship: SUCCESS
update_pirate_treasure: FAILURE
treason: FAILURE
add_pirate: SUCCESS
get_treasure: SUCCESS, 6
get_cannons: SUCCESS, 23
add_pirate: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
add_ship: SUCCESS
add_pirate: FAILURE
remove_pirate: FAILURE
add_ship: SUCCESS
treason: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
update_pirate_treasure: FAILURE
get_cannons: FAILURE
update_pirate_treasure: SUCCESS
update_pirate_treasure: FAILURE
update_pirate_treasure: FAILURE
add_pirate: SUCCESS
ships_battle: FAILURE
remove_pirate: FAILURE
get_cannons: SUCCESS, 3
treason: FAILURE
remove_ship: SUCCESS
add_pirate: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
remove_pirate: INVALID_INPUT
remove_pirate: SUCCESS
add_pirate: SUCCESS
ships_battle: FAILURE
get_richest_pirate: SUCCESS, 358
add_ship: SUCCESS
add_ship: SUCCESS
treason: FAILURE
add_pirate: SUCCESS
get_treasure: SUCCESS, -1
get_cannons: SUCCESS, 18
update_pirate_treasure: FAILURE
ships_battle: SUCCESS
ships_battle: FAILURE
add_ship: SUCCESS
treason: FAILURE
get_richest_pirate: SUCCESS, 184
remove_ship: SUCCESS
remove_ship: SUCCESS
get_cannons: FAILURE
get_cannons: SUCCESS, 14
update_pirate_treasure: SUCCESS
remove_pirate: FAILURE
treason: SUCCESS
get_cannons: SUCCESS, 30
remove_ship: SUCCESS
get_richest_pirate: SUCCESS, 107
ships_battle: SUCCESS
add_pirate: SUCCESS
ships_battle: SUCCESS
add_ship: SUCCESS
remove_pirate: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
ships_battle: SUCCESS
get_treasure: SUCCESS, -1
get_treasure: SUCCESS, 0
add_ship: SUCCESS
remove_pirate: FAILURE
add_pirate: FAILURE
ships_battle: SUCCESS
remove_pirate: FAILURE
get_cannons: SUCCESS, 18
add_pirate: SUCCESS
add_ship: SUCCESS
get_cannons: FAILURE
add_ship: SUCCESS
update_pirate_treasure: FAILURE
ships_battle: SUCCESS
get_treasure: SUCCESS, 3
add_ship: SUCCESS
get_cannons: FAILURE
remove_pirate: SUCCESS
add_pirate: SUCCESS
add_pirate: SUCCESS
ships_battle: SUCCESS
get_cannons: SUCCESS, 22
get_cannons: SUCCESS, 8
treason: FAILURE
get_cannons: SUCCESS, 20
get_cannons: SUCCESS, 22
remove_pirate: FAILURE
add_pirate: SUCCESS
get_cannons: SUCCESS, 22
add_pirate: SUCCESS
get_cannons: FAILURE
remove_ship: SUCCESS
add_ship: SUCCESS
get_treasure: FAILURE
ships_battle: SUCCESS
add_pirate: SUCCESS
update_pirate_treasure: SUCCESS
add_ship: FAILURE
ships_battle: SUCCESS
get_richest_pirate: SUCCESS, 316
get_cannons: SUCCESS, 8
add_ship: SUCCESS
remove_pirate: FAILURE
get_richest_pirate: SUCCESS, 72
add_ship: SUCCESS
get_cannons: SUCCESS, 26
add_pirate: SUCCESS
treason: FAILURE
remove_pirate: FAILURE
get_treasure: SUCCESS, 17
remove_ship: SUCCESS
get_cannons: SUCCESS, 7
remove_ship: FAILURE
get_richest_pirate: FAILURE
add_ship: SUCCESS
update_pirate_treasure: FAILURE
treason: SUCCESS
get_richest_pirate: FAILURE
get_treasure: FAILURE
remove_ship: SUCCESS
get_richest_pirate: SUCCESS, 300
remove_ship: FAILURE
remove_ship: SUCCESS
get_richest_pirate: FAILURE
update_pirate_treasure: FAILURE
get_cannons: FAILURE
remove_pirate: FAILURE
ships_battle: FAILURE
add_ship: SUCCESS
get_treasure: SUCCESS, 0
remove_ship: FAILURE
treason: SUCCESS
get_richest_pirate: SUCCESS, 305
update_pirate_treasure: FAILURE
ships_battle: FAILURE
ships_battle: SUCCESS
update_pirate_treasure: SUCCESS
add_ship: SUCCESS
ships_battle: FAILURE
add_ship: SUCCESS
update_pirate_treasure: FAILURE
treason: SUCCESS
remove_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
get_treasure: FAILURE
treason: FAILURE
remove_pirate: FAILURE
treason: FAILURE
remove_pirate: FAILURE
add_ship: SUCCESS
get_cannons: SUCCESS, 28
get_treasure: SUCCESS, 21
get_richest_pirate: SUCCESS, 356
remove_pirate: SUCCESS
get_cannons: SUCCESS, 26
ships_battle: FAILURE
update_pirate_treasure: FAILURE
treason: SUCCESS
remove_ship: SUCCESS
get_richest_pirate: FAILURE
get_treasure: SUCCESS, 12
treason: FAILURE
add_pirate: SUCCESS
treason: FAILURE
remove_ship: FAILURE
get_richest_pirate: FAILURE
add_ship: SUCCESS
get_richest_pirate: FAILURE
get_richest_pirate: FAILURE
get_treasure: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
update_pirate_treasure: FAILURE
get_cannons: SUCCESS, 6
ships_battle: FAILURE
treason: FAILURE
get_treasure: SUCCESS, 12
get_treasure: FAILURE
update_pirate_treasure: SUCCESS
ships_battle: SUCCESS
get_cannons: SUCCESS, 23
add_ship: SUCCESS
treason: SUCCESS
get_richest_pirate: FAILURE
get_richest_pirate: SUCCESS, 72
get_treasure: FAILURE
remove_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
update_pirate_treasure: SUCCESS
update_pirate_treasure: FAILURE
remove_ship: FAILURE
get_cannons: SUCCESS, 2
treason: FAILURE
update_pirate_treasure: FAILURE
ships_battle: FAILURE
get_cannons: SUCCESS, 15
treason: SUCCESS
get_cannons: SUCCESS, 5
update_pirate_treasure: SUCCESS
get_cannons: SUCCESS, 13
add_ship: SUCCESS
get_cannons: SUCCESS, 16
update_pirate_treasure: FAILURE
update_pirate_treasure: SUCCESS
add_ship: SUCCESS
remove_ship: FAILURE
add_pirate: SUCCESS
add_ship: SUCCESS
remove_ship: FAILURE
remove_pirate: SUCCESS
get_cannons: SUCCESS, 4
add_pirate: SUCCESS
ships_battle: FAILURE
get_treasure: FAILURE
add_ship: SUCCESS
add_pirate: FAILURE
remove_ship: FAILURE
add_pirate: SUCCESS
get_treasure: SUCCESS, 27
get_cannons: SUCCESS, 26
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
update_pirate_treasure: FAILURE
remove_pirate: SUCCESS
remove_pirate: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
get_cannons: SUCCESS, 8
get_treasure: SUCCESS, 0
update_pirate_treasure: FAILURE
remove_ship: FAILURE
remove_pirate: FAILURE
add_pirate: SUCCESS
add_pirate: SUCCESS
get_cannons: SUCCESS, 26
add_ship: SUCCESS
add_ship: FAILURE
get_richest_pirate: SUCCESS, 296
remove_ship: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
get_cannons: FAILURE
ships_battle: SUCCESS
add_pirate: SUCCESS
get_treasure: SUCCESS, 16
ships_battle: SUCCESS
get_cannons: SUCCESS, 14
remove_pirate: SUCCESS
treason: SUCCESS
treason: FAILURE
add_ship: SUCCESS
ships_battle: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
ships_battle: SUCCESS
remove_ship: SUCCESS
get_richest_pirate: SUCCESS, 3
remove_ship: SUCCESS
get_treasure: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
treason: FAILURE
add_pirate: SUCCESS
update_pirate_treasure: FAILURE
remove_pirate: SUCCESS
treason: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
get_cannons: FAILURE
remove_pirate: SUCCESS
get_treasure: SUCCESS, 17
update_pirate_treasure: SUCCESS
ships_battle: FAILURE
remove_ship: SUCCESS
remove_ship: FAILURE
get_cannons: SUCCESS, 0
get_treasure: FAILURE
add_ship: SUCCESS
get_treasure: FAILURE
get_cannons: SUCCESS, 11
add_ship: SUCCESS
add_ship: SUCCESS
get_richest_pirate: FAILURE
get_treasure: SUCCESS, 28
get_treasure: SUCCESS, 24
get_richest_pirate: FAILURE
get_treasure: SUCCESS, 38
add_ship: SUCCESS
ships_battle: SUCCESS
update_pirate_treasure: FAILURE
remove_ship: FAILURE
add_ship: SUCCESS
update_pirate_treasure: SUCCESS
get_cannons: SUCCESS, 24
treason: SUCCESS
ships_battle: SUCCESS
add_ship: SUCCESS
remove_ship: FAILURE
get_treasure: FAILURE
add_ship: SUCCESS
get_treasure: FAILURE
ships_battle: SUCCESS
remove_ship: SUCCESS
ships_battle: SUCCESS
get_cannons: SUCCESS, 8
remove_ship: SUCCESS
get_treasure: SUCCESS, 23
remove_ship: SUCCESS
add_ship: SUCCESS
remove_pirate: FAILURE
add_ship: SUCCESS
get_richest_pirate: FAILURE
remove_pirate: FAILURE
ships_battle: FAILURE
get_richest_pirate: SUCCESS, 267
add_ship: SUCCESS
get_cannons: SUCCESS, 7
add_pirate: SUCCESS
get_treasure: FAILURE
update_pirate_treasure: SUCCESS
get_richest_pirate: FAILURE
add_ship: SUCCESS
get_cannons: FAILURE
ships_battle: SUCCESS
get_richest_pirate: SUCCESS, 356
get_richest_pirate: FAILURE
add_pirate: SUCCESS
add_pirate: SUCCESS
get_cannons: SUCCESS, 8
add_ship: SUCCESS
remove_pirate: SUCCESS
ships_battle: SUCCESS
get_cannons: SUCCESS, 28
add_ship: SUCCESS
get_cannons: SUCCESS, 5
get_richest_pirate: FAILURE
update_pirate_treasure: FAILURE
add_pirate: SUCCESS
ships_battle: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
ships_battle: FAILURE
remove_pirate: SUCCESS
ships_battle: SUCCESS
add_ship: SUCCESS
add_pirate: SUCCESS
remove_ship: SUCCESS
treason: FAILURE
get_treasure: SUCCESS, 24
ships_battle: SUCCESS
remove_pirate: SUCCESS
remove_ship: SUCCESS
add_ship: SUCCESS
get_cannons: SUCCESS, 21
remove_ship: SUCCESS
remove_ship: SUCCESS
ships_battle: FAILURE
add_pirate: FAILURE
get_treasure: SUCCESS, 27
add_ship: SUCCESS
get_cannons: SUCCESS, 13
add_pirate: FAILURE
treason: FAILURE
add_ship: SUCCESS
add_pirate: SUCCESS
add_pirate: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
get_treasure: FAILURE
add_pirate: FAILURE
get_richest_pirate: FAILURE
get_cannons: SUCCESS, 0
remove_pirate: SUCCESS
get_richest_pirate: SUCCESS, 225
update_pirate_treasure: FAILURE
add_ship: INVALID_INPUT
get_cannons: SUCCESS, 16
add_ship: SUCCESS
add_pirate: SUCCESS
remove_pirate: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
add_pirate: SUCCESS
get_richest_pirate: FAILURE
remove_pirate: FAILURE
update_pirate_treasure: FAILURE
add_pirate: FAILURE
add_ship: SUCCESS
remove_pirate: FAILURE
remove_pirate: FAILURE
get_cannons: SUCCESS, 5
add_ship: SUCCESS
remove_ship: SUCCESS
update_pirate_treasure: SUCCESS
add_ship: SUCCESS
ships_battle: FAILURE
ships_battle: FAILURE
add_ship: SUCCESS
ships_battle: FAILURE
add_pirate: SUCCESS
get_treasure: FAILURE
ships_battle: SUCCESS
get_cannons: SUCCESS, 5
update_pirate_treasure: FAILURE
get_treasure: FAILURE
add_pirate: SUCCESS
add_ship: SUCCESS
get_cannons: SUCCESS, 12
remove_pirate: FAILURE
update_pirate_treasure: SUCCESS
get_richest_pirate: SUCCESS, 50
get_richest_pirate: SUCCESS, 225
add_ship: SUCCESS
add_pirate: SUCCESS
ships_battle: FAILURE
add_ship: SUCCESS
get_treasure: FAILURE
ships_battle: SUCCESS
update_pirate_treasure: SUCCESS
get_richest_pirate: SUCCESS, 97
add_ship: SUCCESS
add_pirate: FAILURE
get_cannons: SUCCESS, 0
remove_pirate: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
add_ship: SUCCESS
remove_ship: SUCCESS
add_ship: SUCCESS
remove_ship: FAILURE
add_ship: SUCCESS
get_cannons: SUCCESS, 10
treason: FAILURE
get_cannons: SUCCESS, 6
add_pirate: FAILURE
add_ship: SUCCESS
ships_battle: FAILURE
ships_battle: FAILURE
remove_ship: SUCCESS
add_ship: SUCCESS
ships_battle: SUCCESS
remove_pirate: SUCCESS
add_pirate: SUCCESS
update_pirate_treasure: FAILURE
add_pirate: SUCCESS
add_ship: SUCCESS
get_cannons: SUCCESS, 2
treason: SUCCESS
treason: FAILURE
add_pirate: SUCCESS
get_cannons: SUCCESS, 10
remove_pirate: SUCCESS
get_cannons: FAILURE
add_ship: SUCCESS
add_ship: INVALID_INPUT
update_pirate_treasure: FAILURE
remove_ship: FAILURE
ships_battle: FAILURE
remove_pirate: FAILURE
remove_pirate: FAILURE
get_cannons: SUCCESS, 10
get_cannons: SUCCESS, 7
add_ship: SUCCESS
remove_pirate: FAILURE
get_treasure: SUCCESS, 6
ships_battle: SUCCESS
get_treasure: FAILURE
